### PR TITLE
Refactor `ContractCallEvmCodesTest`

### PR DESCRIPTION
**Pull request by fchitakova**
**Description**:

The idea of this change is to make the test independent of ContractCallTestSetup class and only reuse serviceParametersForExecution method from the base class.

**Related issue(s)**: https://github.com/hashgraph/hedera-mirror-node/issues/8450 

Fixes #

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [X] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java [hunk 1]
```diff
@@ -16,56 +16,293 @@
...
+                .customize(c -> c.contractId(addressThisContractEntityId.getId())
+                        .slot(Bytes.fromHexString("0x0000000000000000000000000000000000000000000000000000000000000000")
+                                .toArrayUnsafe())
+                        .value(Bytes.fromHexString("0x4746573740000000000000000000000000000000000000000000000000000000")
+                                .toArrayUnsafe()))
+                .persist();
+
+        domainBuilder
+                .recordFile()
+                .customize(f -> f.bytes(addressThisContractBytes))
```
[the remark this answers is not in the document]
Comment (fchitakova) [reply hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java]: Removed, thanks!

File: hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java [hunk 2]
```diff
@@ -16,56 +16,293 @@
...
+import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 
-class ContractCallEvmCodesTest extends ContractCallTestSetup {
+class ContractCallEvmCodesTest extends Web3IntegrationTest {
```
Comment (xin-hedera) [inline hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java:69]:
```suggestion
@RequiredArgsConstructor
class ContractCallEvmCodesTest extends Web3IntegrationTest {
```

Then remove all `@Autowired` and change the modifier to `private final`.

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java (hunk 2 above)
Comment (fchitakova) [reply hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java:70]: Done, thanks

File: hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java [hunk 3]
```diff
@@ -173,11 +264,12 @@ void getBlockHashReturnsCorrectHash() {
     void getGenesisBlockHashReturnsCorrectBlock() {
         // Persist all entities so that we can get a specific record file hash and pass it to
         // functionEncodeDecoder#functionHashFor.
```
Comment (IvanKavaldzhiev) [inline hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java:266]: Probably this comment is no longer applicable

**Review by IvanKavaldzhiev — APPROVED** [review]
LGTM, some methods and components will be common for more tests, but can be extracted as a final stage in a separate PR

File: hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java (hunk 3 above)
Comment (fchitakova) [reply hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java]: It is removed, thanks!

**Comment by steven-sheehy** [comment]
Closing in favor of the approach in #8385

**Pull request closed without merging**